### PR TITLE
Hot fix

### PR DESCRIPTION
**Pull request by sojanpr**
Hot fix for
1. App crash on clicking link with no Monster prams
2.Monster is not shown on clicking a link with no
`$android_deeplink_path`

@aaustin @EvangelosG

### CODE REVIEW

File: app/src/main/java/io/branch/branchster/SplashActivity.java [hunk 1]
```diff
@@ -58,10 +59,16 @@ public void onInitFinished(BranchUniversalObject branchUniversalObject, LinkProp
                 if (branchUniversalObject == null) {
                     proceedToAppTransparent();
                 }
+                /* In case the clicked link has $android_deeplink_path the Branch will launch the MonsterViewer automatically since AutoDeeplinking feature is enabled.
+                 * Launch Monster viewer activity if a link clicked without $android_deeplink_path
+                 */
                 else if (!branchUniversalObject.getMetadata().containsKey("$android_deeplink_path")) {
```
Comment (aaustin) [inline app/src/main/java/io/branch/branchster/SplashActivity.java]: Doesn't this mean that deferred deep linking (through install) won't work unless $android_deeplink_path is specified?

Comment (sojanpr) [inline app/src/main/java/io/branch/branchster/SplashActivity.java]: both deferred and non-deferred cases "Splash"  is the intent receiver. SDK Navigates to the Auto deep link activity after opening the splash.  So both cases the redirection happens from Splash

Comment (aaustin) [inline app/src/main/java/io/branch/branchster/SplashActivity.java]: But the auto deep link key is for a deep link path: `<meta-data android:name="io.branch.sdk.auto_link_path" android:value="monster/view/" />`. Just to make it clear, marketing links and iOS created links don't have deep link path. Here's one to try with: bnc.lt/ALMc/o6F0V0xU7u

Comment (aaustin) [inline app/src/main/java/io/branch/branchster/SplashActivity.java]: I'd recommend making the auto deep link based on keys for `monster`

Comment (sojanpr) [inline app/src/main/java/io/branch/branchster/SplashActivity.java]: The reason of the  above link not opens is with incorrect link dat ( link has `clicked_branch_link` as 1 while expecting true /false) . The links created from iOS are working fine

Comment (aaustin) [inline app/src/main/java/io/branch/branchster/SplashActivity.java]: The live version of iOS is not putting deep link path keys in the links. Here's another one that doesn't have `+clicked_branch_link` erroneously put in the link https://bnc.lt/ALMc/iLhneh1Qav. Again, I'd recommend that we change the deep link key to `monster`.

Comment (aaustin) [inline app/src/main/java/io/branch/branchster/SplashActivity.java]: Sorry for misreading this. Totally my fault.

**Comment by aaustin** [comment]
👍

**Pull request merged**